src/app/models/config/acknowledgements.config.ts

```diff
@@ -0,0 +1,3 @@
+export const marks = ['bold', 'italic', 'link', 'subscript', 'superscript'];
```

Comment:
If I create a link, it doesn't display the 'edit link' popup when I select it.

Reply:
we have only implemented links for Rich text inputs. At the moment it doesn't work in manuscript sections yet

Reply:
Ok, just need to make sure that is understood by production and is planned to be implemented during the body text.